src/ko/validation-inline.php

```diff
@@ -13,110 +13,110 @@
+        'file'    => '이 파일의 용량은 :value킬로바이트보다 같거나 작아야 합니다.',
+        'string'  => '이 문자열의 길이는 :value보다 같거나 짧아야 합니다.',
+        'array'   => '이 배열의 항목 수는 :value개 보다 같거나 작아야 합니다.',
     ],
     'max' => [
-        'numeric' => 'The value may not be greater than :max.',
-        'file'    => 'The file size may not be greater than :max kilobytes.',
-        'string'  => 'The string may not be greater than :max characters.',
-        'array'   => 'The content may not have more than :max items.',
+        'numeric' => '이 값은 :max보다 클 수 없습니다.',
+        'file'    => '이 파일의 용량은 :max킬로바이트보다 클 수 없습니다.',
+        'string'  => '이 문자열의 길이는 :max자보다 클 수 없습니다.',
+        'array'   => '이 배열의 항목 수는 :max개보다 많을 수 없습니다.',
     ],
-    'mimes'     => 'This must be a file of type: :values.',
-    'mimetypes' => 'This must be a file of type: :values.',
+    'mimes'     => '다음의 파일 형식이어야 합니다: :values.',
+    'mimetypes' => '다음의 파일 형식이어야 합니다: :values.',
     'min'       => [
-        'numeric' => 'The value must be at least :min.',
-        'file'    => 'The file size must be at least :min kilobytes.',
-        'string'  => 'The string must be at least :min characters.',
-        'array'   => 'The value must have at least :min items.',
+        'numeric' => '이 값은 최소한 :min이어야 합니다.',
+        'file'    => '이 파일의 용량은 최소한 :min킬로바이트이어야 합니다.',
+        'string'  => '이 문자열의 길이는 최소한 :min자이어야 합니다.',
+        'array'   => '이 배열의 항목 수는 최소한 :min개의 항목이 있어야 합니다.',
     ],
-    'not_in'               => 'The selected value is invalid.',
-    'not_regex'            => 'This format is invalid.',
-    'numeric'              => 'This must be a number.',
-    'password'             => 'The password is incorrect.',
-    'present'              => 'This field must be present.',
-    'regex'                => 'This format is invalid.',
-    'required'             => 'This field is required.',
-    'required_if'          => 'This field is required when :other is :value.',
-    'required_unless'      => 'This field is required unless :other is in :values.',
-    'required_with'        => 'This field is required when :values is present.',
-    'required_with_all'    => 'This field is required when :values are present.',
-    'required_without'     => 'This field is required when :values is not present.',
-    'required_without_all' => 'This field is required when none of :values are present.',
-    'same'                 => 'The value of this field must match the one from :other.',
+    'not_in'               => '선택된 값은 올바르지 않습니다.',
+    'not_regex'            => '형식이 올바르지 않습니다.',
+    'numeric'              => '숫자여야 합니다.',
+    'password'             => '비밀번호가 잘못되었습니다.',
+    'present'              => '이 필드가 있어야 합니다.',
+    'regex'                => '형식이 올바르지 않습니다.',
+    'required'             => '이 필드는 필수입니다.',
+    'required_if'          => ':other이(가) :value 일 때 이 필드는 필수입니다.',
+    'required_unless'      => ':other이(가) :values에 없다면 :attribute 이 필드는 필수입니다.',
```

Comment:
the problem is here ( sorry I don't validate the review )

Reply:
No :attribute in validation-inline.php

Reply:
thank you 
i'll fix it